src/types/uint-16.ts

```diff
@@ -5,11 +5,13 @@ import { BinaryParser } from "../serdes/binary-parser";
  * Derived UInt class for serializing/deserializing 16 bit UInt
  */
 class UInt16 extends UInt {
-  protected static readonly width: number = 16 / 8 //2
-  static readonly defaultUInt16: UInt16 = new UInt16(Buffer.alloc(UInt16.width))
+  protected static readonly width: number = 16 / 8; //2
```

Comment:
Space after comment:

```suggestion
  protected static readonly width: number = 16 / 8; // 2
```

Reply:
Unless this is a linter reverting it?

Reply:
It was the linter that made the changes to `UInt` classes.